app.rb

```diff
@@ -1,7 +1,34 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/player'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
```

Comment:
good to see the utilisation of  post , get , and redirect routes in the right way